filebeat/module/kafka/_meta/kibana/6/dashboard/Filebeat-Kafka-overview.json

```diff
@@ -117,11 +117,11 @@
                             }
                         ]
                     }, 
-                    "title": "Number of Kafka stracktraces by class", 
+                    "title": "Number of Kafka stracktraces by class 7.x",
                     "type": "histogram"
                 }
             }, 
-            "id": "Number of Kafka stracktraces by class", 
+            "id": "number-of-kafka-stracktraces-by-class-7x", 
```

Comment:
@jsoriano I modified this id here manually as it seemed odd that it is identical to a title (and contains spaces)

Reply:
I added a separate PR for this change here to be able to keep resetting this without having to apply this change manually: https://github.com/elastic/beats/pull/10067

Reply:
You can also add the missing `[Filebeat Kafka]` suffix to the title.

Reply:
@jsoriano Could you open a separate PR for this? The reason I do not want to do any manual changes in this PR is because I keep resetting the changes if PR's are merged and reapply the script. If I have manual changes in here, it would mean I have to keep track of these changes and reapply them every time.

Reply:
I meant that the change of the id wouldn't be needed with the suffix in the title :slightly_smiling_face: 

I have created a PR to fix this title in any case https://github.com/elastic/beats/pull/10107